Neos.Neos/Classes/Controller/Module/Administration/PackagesController.php

```diff
@@ -30,13 +31,15 @@ class PackagesController extends AbstractModuleController
     /**
      * @return void
      */
-    public function indexAction()
+    public function indexAction(string $sortBy = 'name', string $sortDirection = QueryInterface::ORDER_ASCENDING)
```

Comment:
I suggest to avoid introducing this dependency to the Persistence submodule

```suggestion
    public function indexAction(string $sortBy = 'name', bool $descending = false)
```